### PR TITLE
Added ports that need to be open

### LINKED ISSUE #18: Add network and firewall requirements
please add network and firewall requirements to the documentation?

In case OpenSearch is used in production on-premise, it would be great to get an overview of all the network requirements eg. firewall ports that needs to be opened.

Below a list of some ports (might not be complete) :

Firewall Ports
port: comment
443: OpenSearch Dashboards
5601: OpenSearch Dashboards
9000: Cerebro
9200: REST
9250: cross cluster search
9300: nodes communication / transport
9600: required for Performance Analyzer

Original issue here: https://github.com/opendistro/for-elasticsearch-docs/issues/433

### PR DESCRIPTION
**Pull request by kolchfa-aws**
Fixes #18 

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _opensearch/install/important-settings.md [hunk 1]
```diff
@@ -38,3 +38,17 @@ The [sample docker-compose.yml]({{site.url}}{{site.baseurl}}/opensearch/install/
...
 
   Allows you to access Performance Analyzer on port 9600.
+
+### Network requirements
+
+  The following firewall ports need to be open for OpenSearch components.
+
+Port number | OpenSearch component
+:--- | :--- 
+443 | OpenSearch Dashboards
```
Comment (alicejw-aws) [inline _opensearch/install/important-settings.md:48]: Just a thought. . if both 443 and 5601 are required to use the Dashboards in all cases, maybe indicate that both are required. If not, perhaps indicate when they would need both. (e.g. optionally both are required if <use case or condition> exists).

**Review by alicejw-aws — APPROVED** [review]
Looks good!

**Review by hdhalter — APPROVED** [review]

File: _opensearch/install/important-settings.md (hunk 1 above)
Comment (Naarcha-AWS) [reply _opensearch/install/important-settings.md:48]: Remove "firewall".

**Review by Naarcha-AWS — APPROVED** [review]

File: _opensearch/install/important-settings.md (hunk 1 above)
Comment (kolchfa-aws) [reply _opensearch/install/important-settings.md]: Added distinction between 443 and 5601

**Pull request merged**